src/main/kotlin/com/hyperdevs/poeditor/gradle/tasks/ImportPoEditorStringsTask.kt

```diff
@@ -45,8 +45,9 @@ abstract class ImportPoEditorStringsTask
         val projectId: Int
         val defaultLang: String
         val defaultResPath: String
-        val tags : List<String>
+        val tags: List<String>
         val languageOverridePathMap: Map<String, String>
+        val minimumTranslationPercentage: Int?
```

Comment:
If you apply the suggestion that I mentioned in `PoEditorPlugin`, you can make this non-nullable